example/.lein-classpath

```diff
@@ -1 +0,0 @@
-../src
```

Comment:
Did you intend to remove this file?

This was added explicitly so you can dev/test the plugin without having to `lein install` it everytime.  [See here](https://github.com/technomancy/leiningen/blob/stable/doc/PLUGINS.md#local-development).  

Perhaps its presence means it's less of an `example` project though, and more a project for development/testing.

What do you think?

Reply:
Ah I understand now! And, as you mention, this is not an `example` project but a test project in my opinion. I have to think a bit more and propose something that makes sense. I will update this PR to remove only the `.lein-repl-history` file

Reply:
Yeah.  My intention was for it to serve as both, rather than having to maintain two.